internal/services/worker_version/schema.go

```diff
@@ -254,9 +255,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 			},
 			"assets": schema.SingleNestedAttribute{
 				Description: "Configuration for assets within a Worker.\n\n[`_headers`](https://developers.cloudflare.com/workers/static-assets/headers/#custom-headers) and\n[`_redirects`](https://developers.cloudflare.com/workers/static-assets/redirects/) files should be\nincluded as modules named `_headers` and `_redirects` with content type `text/plain`.",
-				Computed:    true,
 				Optional:    true,
-				CustomType:  customfield.NewNestedObjectType[WorkerVersionAssetsModel](ctx),
```

Comment:
`assets` recently became codegen'd as `Computed: true` (and switched to using a custom type to handle this) because we made the nested `assets.config.*` attributes have defaults. But the top-level `assets` property isn't always returned by the API (i.e. if your Version doesn't include any assets) so bubbling up the `Computed: true` to the top-level `assets` attribute was wrong.